myia/public_api.py

```diff
@@ -21,13 +21,13 @@
 import numpy as np
 import torch
```

Comment:
If this is to become the public myia API it can't have a dependency on PyTorch.

As it clearly still does, I don't approve of the rename.